install_config/configuring_admission_control.adoc

```diff
@@ -0,0 +1,161 @@
+
+== Configuring the list of admission control plugins
+
+The OpenShift master uses two sets of admission control plugins. One set controls admission of Kubernetes-specific
+resources such as Pods, ReplicationControllers, Services, etc. and another one controls admission of OpenShift-specific resources such
+as Builds, Projects, Deployments, etc.
+
+[WARNING]
+====
+The OpenShift master has a default list of plugins that are enabled by default for each type of resource (Kubernetes and OpenShift).
+These are required for the proper functioning of the master. It is not recommended to modify these lists unless you strictly know what
+you are doing. Future versions of the product may use a different set of plugins and may change their ordering. If you do override the
+default list of plugins in the master config, you are responsible for updating it to reflect requirements of newer versions of the
+OpenShift master.
+====
+
+=== Overriding the list of admission control plugins for OpenShift resources
+
+The list of admission control plugins for OpenShift resources can be configured in the
+link:master_node_configuration.html#master-configuration-files[master-config.yaml]
+as follows:
+
+[source,yaml]
+----
+admissionControl:
+  pluginOrderOverride:
+  - OriginNamespaceLifecycle
+  - BuildByStrategy
+  - YourPlugin
+----
+
+If the `pluginOrderOverride` setting is empty or not present, then the default set of plugins will be used.
+
+The default list of admission control plugins for OpenShift resources is:
+
+[source, yaml]
+----
+- OriginNamespaceLifecycle
+- BuildByStrategy
+----
+
+* *OriginNamespaceLifecycle* Enforces that a
+link:../architecture/core_concepts/projects_and_users.html#namespaces[Namespace]
+must exist before creating resources in it. It also prevents creation of resources in a project
+that is in the process of getting deleted.
+
+* *BuildByStrategy* Restricts creation of
+link:../architecture/core_concepts/builds_and_image_streams.html#builds[Builds]
+and
+link:../dev_guide/builds.html#defining-a-buildconfig[BuildConfigs]
+to a strategy that the current user is
+link:../admin_guide/securing_builds.html[allowed to use].
+
+=== Overriding the list of admission control plugins for Kubernetes resources
+
+The list of admission control plugins for Kubernetes resources can be configured in the
+link:master_node_configuration.html#master-configuration-files[master-config.yaml]
+as follows:
+
+[source,yaml]
+----
+kubernetesMasterConfig:
+  admissionConfig:
+    pluginOrderOverride:
+    - NamespaceLifecycle
+    - OriginPodNodeEnvironment
+    - YourPlugin
+    - ...
+----
+
+The default list of admission control plugins for Kubernetes resources is:
+
+[source, yaml]
+----
+- NamespaceLifecycle
+- OriginPodNodeEnvironment
+- LimitRanger
+- ServiceAccount
+- SecurityContextConstraint
+- ResourceQuota
+- SCCExecRestrictions
```

Comment:
It's not in the code. We should open an issue. 
https://github.com/openshift/origin/blob/master/pkg/cmd/server/kubernetes/master_config.go#L38